tests/test_api_build.py

```diff
@@ -1971,6 +1971,16 @@ def test_add_pip_as_python_dependency_from_condarc_file(
             api.build(testing_metadata)
 
 
+@pytest.mark.skipif(on_win, reason="Tests cross-compilation targeting Windows")
+def test_cross_unix_windows_mingw(testing_config):
+    recipe = os.path.join(metadata_dir, "_cross_unix_windows_mingw")
+    testing_config.channel_urls = [
+        "conda-forge/label/m2w64-experimental",
+        "conda-forge",
```

Comment:
We might need adjustments further down the line, e.g., no idea whether `conda-forge/label/m2w64-experimental` will stay and could make the test more self-contained.
But I think it is fine to include this as is for now!